.github/workflows/main.yml

```
@@ -35,4 +35,4 @@ jobs:
       - name: Get PDFs
         uses: actions/download-artifact@v3
       - name: Run VeraPDF
-        run: verify Thesis/*.pdf | tee /dev/stderr | grep -qE 'nonCompliant="0"'
+        run: verify Thesis/*.pdf | tee /dev/stderr | grep -qE 'nonCompliant="0" failedJobs="0"'
```

Hi,
thanks for noticing this :D I didn't really check if it fails properly.

Since the original problem is that the pipeline discards the exit status of `verify` and there's no point in looking at the grepped contents, shouldn't this work better?
```suggestion
        run: verify Thesis/*.pdf
```
This ofc assumes that the verify script exits properly, which I see is not the case...so this might require more fixing in https://github.com/mff-cuni-cz/cuni-thesis-validator/blob/master/verify#L3 as well:

```diff
-verapdf --profile /opt/uk-profile.xml $@
+exec verapdf --profile /opt/uk-profile.xml "$@"
```

if you see that viable can you please send a MR in that direction too? If not, lmk, we can just merge this so that the issue is solved and elaborate later. :)

Thanks!

---

(on the second thought, the immediate fix is OK, merging :D )